rocca/agents/utils.py

```diff
@@ -34,7 +34,7 @@
 # Variables #
 #############
 
-agent_log = create_logger("opencog.log")
+agent_log = create_logger("opencog_agent.log")
```

Comment:
Note that the fact that all loggers point to the same destination file is done purposely.  I find it makes it easier to understand the processing flow between the different parts.  For instance if there is a bug in the miner, having the miner messages logged right after the agent's allows me to understand whether the calls of the miner were correct to begin with, etc.

If afterward I want to split that log file per component I can use

https://github.com/opencog/cogutil/blob/master/scripts/util/filter-log.sh

Writing multiple log files might be an option if we have a tool to merge multiple log files into one and properly interleaving the messages to respect chronological order.  If you wish to add such a tool so you can keep having different destination log files, then I don't mind.

Reply:
Okay, I will bring it back. To be honest, the sheer volume of the logs makes them pretty much useless to me anyway.

Reply:
Done